test/index.spec.js

```diff
@@ -1,12 +1,5 @@
 import test from 'tape';
-import {
-  tileToCell,
-  cellToTile,
-  cellToParent,
-  geometryToCells,
-  getResolution,
-  hexToBigInt
-} from 'quadbin';
+import {tileToCell, cellToTile, cellToParent, geometryToCells, getResolution} from 'quadbin';
```

Comment:
Could you add a few test cases, including near the antimeridian?

Reply:
Let me give it a try, sure!

Reply:
Done! 🤞